config/nova-pages-tool.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'languages' => [
+        'en' => 'English',
+        'fr' => 'Français',
+        'nl' => 'Nederlands',
+    ],
+    'defaultLanguage' => 'en',
```

Comment:
Hoe verhoudt deze variabele zich tot Laravel configs `app.locale` en `app.fallback_locale`?

Reply:
Je kunt in Laravel volgens mij geen config waardes uitlezen in andere config bestanden.
Dan gaan er dingen fout in de categorie recursie.

Maar ik ga deze config file publiceerbaar maken en dan mogen developers dit vullen zoals ze maar willen, met dezelfde waarde als `app.locale` of een ander.